### DISCUSSION TITLE
minicom: dependency on lrzsz for ymodem/zmodem

### DISCUSSION THREAD

**Discussion by tewarid** [Homebrew/discussions#3699] [Everyday usage]
I tried using minicom to send files using ymodem but it didn't work at first because the program used to send files (`sb`) wasn't installed. I had to install `lrzsz` and then symlink `sb` to `sz` for it to work.

Suggestions
- There should be a dependency on `lrzsz` package in `minicom`
- `lrzsz` should create `sb` symlink to `sz`

**Comment by SMillerDev** [comment]
Please create pull requests to add those.

**Comment by tewarid** [comment]
I'm unable to test updates to formulae.

Doing something like `brew reinstall --build-from-source --verbose --debug minicom` fails with

```log
...
==> Reinstalling minicom 
Error: Your Command Line Tools (CLT) does not support macOS 13.
It is either outdated or was modified.
Please update your Command Line Tools (CLT) or delete it if no updates are available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 14.0.
```

I've followed the suggested fixes without any success. I'm on latest macOS Ventura beta (13.0) and Xcode is up to date (14.0.1).

**Reply by SMillerDev** [reply]
Yeah, you need to download the beta CLT from the Apple developer portal.

**Reply by tewarid** [reply]
Interesting - I wasn't aware you had to download and install beta command lines tools manually, doing so resolved the build issue.

**Reply by SMillerDev** [reply]
Sometimes you do, sometimes you don't. But if you have commandline tools for a released xcode you should probably install the beta manually.

**Reply by tewarid** [reply]
Thanks - submitted PR https://github.com/Homebrew/homebrew-core/pull/111852.

**Reply by gmrock** [reply]
@tewarid @SMillerDev  - any clue why the PR wasn't merged?

**Reply by tewarid** [reply]
I didn't get a response to comments, nor have an urgent need, or the time to keep bumping it from going stale :)

**Reply by gmrock** [reply]
Thanks @tewarid . @SMillerDev hopefully it's merged. 

Took sometime to figure our why minicom worked flawlessly on Linux but wasn't able to send file(XMODEM - sx) on Mac (installed via homebrew).
Until that timeI have made local changes.
